README.md

```diff
@@ -157,10 +157,12 @@ By default, suspenders will:
 * Create `staging` and `production` branches for deployment
 * Make an initial commit
 
+By default, pre-commit hooks are setup to run `bin/rails`. To override this while committing pass the `--no-verify` option to your commit.
```

Comment:
I think you mean `bin/rake` ?

Reply:
Can you also add the `no-verify` note to the README template (the one that gets copied into the newly created Rails app)